### PR TITLE
Update resource recommendation guide for Promscale

### PR DESCRIPTION
**Pull request by VineethReddy02**
# Description

Added Prometheus `remote_write` recommended config and TimescaleDB recommended configuration. 

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Review by Rajakavitha1 — APPROVED** [review]

File: promscale/installation/recomm-guide.md [hunk 1]
```diff
@@ -104,17 +104,26 @@ The default chunk interval is `1h`
...
\ No newline at end of file
+To set the most common parameters to optimal values based on your system run the
+`timescaledb-tune`. It accounts for memory, CPU, and PostgreSQL version. For
+more information, see [configuration][timescale-tune-configuration]. However,
+there are a few other PostgreSQL parameters worth tuning:
+
+* `checkpoint_timeout=15min` - when a lot of data is ingested increase the
+  checkpoint timeout to reduce the input/output pressure.
+* `bgwriter_delay=10ms` - the background writer needs to be active to reduce the
+  delay
```
Comment (Loquacity) [inline promscale/installation/recomm-guide.md:115]:
```suggestion
* `bgwriter_delay=10ms` - the background writer needs to be active to reduce
  delays.
```

File: promscale/installation/recomm-guide.md [hunk 2]
```diff
@@ -104,17 +104,26 @@ The default chunk interval is `1h`
...
+  writer handles to make it more efficient.
+* `max_wal_size=(around 25% of total disk volume)` - the checkpoint is triggered
+  by the timeout setting and not when the `maximum_wal_size` is reached.
+* `synchronous_commit=off` - this does not cause data corruption or
+  inconsistency. However, in case of a crash, some of the last data points may be
+  lost. For a monitoring observability use case, it’s a reasonable tradeoff to
+  increase ingest performance. 
+
+<highlight type="important"> Make sure that the maximum latency between the
+Promscale connector and the database is no more than `100ms`. </highlight>
```
Comment (Loquacity) [inline promscale/installation/recomm-guide.md:126]:
```suggestion
<highlight type="important"> 
Make sure that the maximum latency between the
Promscale connector and the database is no more than 100ms. 
</highlight>
```

File: promscale/installation/recomm-guide.md [hunk 3]
```diff
@@ -104,17 +104,26 @@ The default chunk interval is `1h`
...
+
+[timescale-tune-configuration]:
+    timescaledb/:currentVersion:/how-to-guides/configuration/timescaledb-tune/
```
Comment (Loquacity) [inline promscale/installation/recomm-guide.md:129]:
```suggestion
[timescale-tune-configuration]: timescaledb/:currentVersion:/how-to-guides/configuration/timescaledb-tune/
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by Loquacity — APPROVED** [review]

**Comment by VineethReddy02** [comment]
closing this in favor to #1093

**Pull request closed without merging**